pkg/volume/emptydir/empty_dir.go

```diff
@@ -271,8 +316,13 @@ func (ed *emptyDir) setupTmpfs(dir string) error {
 		return nil
 	}
 
-	klog.V(3).Infof("pod %v: mounting tmpfs for volume %v", ed.pod.UID, ed.volName)
-	return ed.mounter.Mount("tmpfs", dir, "tmpfs", nil /* options */)
+	var options []string
+	// Linux system default is 50% of capacity.
```

Comment:
what do you mean in this comment?

Reply:
```
size:      The limit of allocated bytes for this tmpfs instance. The 
           default is half of your physical RAM without swap. If you
           oversize your tmpfs instances the machine will deadlock
           since the OOM handler will not be able to free that memory.
```
from: https://www.kernel.org/doc/Documentation/filesystems/tmpfs.txt